### ISSUE TITLE
[BUG] RGB Light Entity not showing light.xyz

### ISSUE THREAD

**Issue by EvBooKY** [Pyhive/HA-Hive-Custom-Component#64]
**Describe the bug**
RGB Lights not behaving as expected. A device for the light exists in my case it is called Study. That device shows two sensors (availability and mode) which appear to be updating. However the light can not be turned off or on. 

![image](https://user-images.githubusercontent.com/65856607/108631990-8dd48980-7464-11eb-8610-42cd3de38ce3.png)

**To Reproduce**
Steps to reproduce the behaviour:
1. Updated to 2021.2
2. Removed Hive restarted
3. Reinstalled Hive 2021.2 
4. In Devices found light and click on it. 
5. In Lovelave light.study is shown but it is unable to be controlled.

**Expected behaviour**
Light can be controlled like other lights including standard Hive Bulbs.

**Screenshots**
See above.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Chrome
 - Version : 88.0.4324.182 

**Additional context**
Other white bulbs including those with warm / cold temp settings are working as expected.

**Comment by KJonline** [comment]
Is your light In a group within the Hive app?

**Comment by EvBooKY** [comment]
Hi, no I removed all the Light Groups previously due to issues. So no it is standalone. The integration seems to see it, but not add it correctly.

**Comment by KJonline** [comment]
Are there any errors in the logs?

**Comment by EvBooKY** [comment]
There are ...

Logger: homeassistant.components.sensor
Source: custom_components/hive/sensor.py:270
Integration: Sensor (documentation, issues)
First occurred: 16:31:47 (1 occurrences)
Last logged: 16:31:47

hive: Error on device update!
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 358, in _async_add_entity
    await entity.async_device_update(warning=False)
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 466, in async_device_update
    await task
  File "/config/custom_components/hive/sensor.py", line 157, in async_update
    self.attributes = await self.get_hotwater_state_sa()
  File "/config/custom_components/hive/sensor.py", line 270, in get_hotwater_state_sa
    snan = await self.hive.hotwater.get_schedule_now_next_later(
  File "/usr/local/lib/python3.8/site-packages/pyhiveapi/hotwater.py", line 135, in get_schedule_now_next_later
    state = await self.helper.getScheduleNNL(
  File "/usr/local/lib/python3.8/site-packages/pyhiveapi/helper/hive_helper.py", line 89, in getScheduleNNL
    date_time_now = datetime.now()
AttributeError: module 'datetime' has no attribute 'now'

**Comment by EvBooKY** [comment]
Plus this, I think the others are all the same

Logger: homeassistant.components.sensor
Source: custom_components/hive/sensor.py:213
Integration: Sensor (documentation, issues)
First occurred: 16:31:47 (12 occurrences)
Last logged: 16:31:47

hive: Error on device update!
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 358, in _async_add_entity
    await entity.async_device_update(warning=False)
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 466, in async_device_update
    await task
  File "/config/custom_components/hive/sensor.py", line 145, in async_update
    self.attributes = await self.get_heating_state_sa()
  File "/config/custom_components/hive/sensor.py", line 213, in get_heating_state_sa
    snan = await self.hive.heating.get_schedule_now_next_later(self.device)
  File "/usr/local/lib/python3.8/site-packages/pyhiveapi/heating.py", line 250, in get_schedule_now_next_later
    state = await self.helper.getScheduleNNL(
  File "/usr/local/lib/python3.8/site-packages/pyhiveapi/helper/hive_helper.py", line 89, in getScheduleNNL
    date_time_now = datetime.now()
AttributeError: module 'datetime' has no attribute 'now'

**Comment by EvBooKY** [comment]
Those errors look more likely to be connected with the other heating and hot water issues. However this maybe the light issue.

2021-02-21 16:31:47 ERROR (MainThread) [homeassistant.components.light] hive: Error on device update!
Traceback (most recent call last):
File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 358, in _async_add_entity
await entity.async_device_update(warning=False)
File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 466, in async_device_update
await task
File "/config/custom_components/hive/light.py", line 160, in async_update
self.device = await self.hive.light.get_light(self.device)
File "/usr/local/lib/python3.8/site-packages/pyhiveapi/light.py", line 59, in get_light
"hs_color": await self.get_color(device),
File "/usr/local/lib/python3.8/site-packages/pyhiveapi/light.py", line 156, in get_color
(data["state"]["hue"]) / 360,
TypeError: unsupported operand type(s) for /: 'NoneType' and 'int'

**Comment by KJonline** [comment]
@EvBooKY what mode is light in when you get this error?

**Comment by EvBooKY** [comment]
I think the light was powered but was off in Hive. I've just restarted HomeAssistant as the light is currently on. This issue has been around for some time and I have noticed that some times the study light is there and others it is not. I wonder if there is a state the light needs to be in when HA is restarted. 
No, doesn't appear to make a difference whether the light is on or off when HA is restarted.
